<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homepage</class>
 <widget class="QMainWindow" name="homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iCardSIS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 225, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>100</width>
      <height>92</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>92</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images/src.qrc">:/images/4.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 24pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>KATHMANDU UNIVERSITY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Dhulikhel,Kavrepalanchowk</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>159</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>ID Card Number:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>541</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="title">
     <string>Personal Information:</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid;
border-radius:10px;
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid;
border-radius:10px;
border-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid;
border-radius:10px;
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Blood Group</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid;
border-radius:10px;
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Date of Birth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid;
border-radius:10px;
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Years</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images/src.qrc">:/images/name.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images/src.qrc">:/images/course.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images/src.qrc">:/images/blood.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images/src.qrc">:/images/year.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images/src.qrc">:/images/cal.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="studentName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="studentDOB">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="studentCourse">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="year">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="bloodGroup">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>330</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_library">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
}

QPushButton:pressed{
	background-color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Library Log</string>
    </property>
    <property name="icon">
     <iconset resource="images/src.qrc">
      <normaloff>:/images/11.jpg</normaloff>:/images/11.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_balance">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
}

QPushButton:pressed{
	background-color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Balance And Enquiry</string>
    </property>
    <property name="icon">
     <iconset resource="images/src.qrc">
      <normaloff>:/images/12.jpg</normaloff>:/images/12.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_transport">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
}

QPushButton:pressed{
	background-color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Transportation Status</string>
    </property>
    <property name="icon">
     <iconset resource="images/src.qrc">
      <normaloff>:/images/13.jpg</normaloff>:/images/13.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_activity">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
}

QPushButton:pressed{
	background-color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Activity</string>
    </property>
    <property name="icon">
     <iconset resource="images/src.qrc">
      <normaloff>:/images/14.jpg</normaloff>:/images/14.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images/src.qrc">:/images/2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>0</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
}

QPushButton:pressed{
	background-color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/src.qrc">
      <normaloff>:/images/10.jpg</normaloff>:/images/10.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>240</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="studentPhoto">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images/src.qrc"/>
 </resources>
 <connections/>
</ui>
